portals/publisher/source/src/app/components/Apis/Details/Policies/PoliciesExpansion.tsx

```diff
@@ -0,0 +1,281 @@
+                //         }
+                //     })();
+            }
+        })
+        setRequestFlowPolicyList(requestFlowList);
+
+        if (!isChoreoConnectEnabled) {
+            // Populate response flow attached policy list
+            const responseFlowList: AttachedPolicy[] = [];
+            const responseFlow = operationInAction.operationPolicies.response;
+            responseFlow.map((responseFlowAttachedPolicy: any) => {
+                const { policyId, policyName, uuid } = responseFlowAttachedPolicy;
+                const policyObj = allPolicies?.find((policy: PolicySpec) => policy.id === policyId)
+                    || allPolicies?.find((policy1: PolicySpec) => policy1.name === policyName);
+                if (policyObj) {
+                    responseFlowList.push({ ...policyObj, uniqueKey: uuid });
+                }
+            })
+            setResponseFlowPolicyList(responseFlowList);
+
+            // Populate fault flow attached policy list
+            const faultFlowList: AttachedPolicy[] = [];
+            const faultFlow = operationInAction.operationPolicies.fault;
+            faultFlow.map((faultFlowAttachedPolicy: any) => {
+                const { policyId, policyName, uuid } = faultFlowAttachedPolicy;
+                const policyObj = allPolicies?.find((policy: PolicySpec) => policy.id === policyId)
+                    || allPolicies?.find((policy1: PolicySpec) => policy1.name === policyName);
+                if (policyObj) {
+                    faultFlowList.push({ ...policyObj, uniqueKey: uuid });
+                }
+            })
+            setFaultFlowPolicyList(faultFlowList);
+        }
+
+    }, [apiOperations])
+
+
+    return (
+        <ExpansionPanelDetails>
+            <Grid spacing={2} container direction='row' justify='flex-start' alignItems='flex-start'>
+                <Grid item xs={12} md={12}>
+                    <Box className={classes.flowSpecificPolicyAttachGrid}>
+                        <Typography variant='subtitle2' align='left'>
+                            <FormattedMessage
+                                id='Apis.Details.Policies.Operation.request.flow.title'
+                                defaultMessage='Request Flow'
+                            />
+                        </Typography>
+                        <FlowArrow arrowDirection='left' />
+                        <PolicyDropzone
+                            policyDisplayStartDirection='left'
+                            currentPolicyList={requestFlowPolicyList}
+                            setCurrentPolicyList={setRequestFlowPolicyList}
+                            droppablePolicyList={requestFlowDroppablePolicyList}
+                            currentFlow='request'
+                            target={target}
+                            verb={verb}
+                            allPolicies={allPolicies}
+                        />
+                    </Box>
+                    {(() => {
```

Comment:
Why do we create a closure here ? 
Creating a function and calling it ? I think this is not needed ?
If you need `if else` conditions, just use JSX syntax with shorthand if-else notation